public/usage-examples/geometry/point_at/point_at-1-simple-oop.cs

```diff
@@ -0,0 +1,43 @@
+using SplashKitSDK;
+
+namespace PointAt
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            SplashKit.OpenWindow("Point At", 800, 600);
```

Comment:
Rather than naming the window the name of the function, try naming it as a description of what is happening like "Face Generator" or something.